FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -20,5 +20,14 @@ public final class RemoteFeedLoader: FeedLoader {
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
 		
+		client.get(from: url) { [weak self] (result) in
+			guard self != nil else { return }
+			switch result {
+			case .success((let data, let response)):
+				completion(Root.feedImagesFromData(from: data, stausCode: response.statusCode))
+			case .failure:
+				completion(.failure(RemoteFeedLoader.Error.connectivity))
```

Comment:
`RemoteFeedLoader` can be omitted from `RemoteFeedLoader.Error.connectivity` since the error type is declared under the `RemoteFeedLoader` namespace.

```suggestion
				completion(.failure(Error.connectivity))
```